src/locales/en.json

```diff
@@ -214,6 +214,10 @@
     "saveAll": {
       "text": "Save All",
       "tooltip": "Download a JSON file of the institutions"
+    },
+    "delete": {
+      "text": "Delete Institution",
```

Comment:
Just "Delete". Move to the more generic button because we can reuse it with panels as well.

Reply:
updated